scripts/eqtl_hail_batch/generate_eqtl_spearman.py

```diff
@@ -536,6 +567,7 @@ def main(
 
     merge_job = batch.new_python_job(name='merge_scatters')
     merge_job.cpu(2)
+    merge_job.image(MULTIPY_IMAGE)
```

Comment:
Small comment would be good for why using the multipy image here

Reply:
added